nf_core/modules/test_yml_builder.py

```diff
@@ -227,20 +227,22 @@ def create_test_file_dict(self, results_dir, is_repeat=False):
         test_files = []
         for root, dir, file in os.walk(results_dir):
             for elem in file:
```

Comment:
Generally dislike the variable names that are used here - a bit unclear and also overwriting the a loop variable is not great practice. Feel free to refactor if you like.

For example, something like this might be clearer / easier to read and understand... (ditch the vague `elem` and don't overwrite it)

```suggestion
        for root, dir, files in os.walk(results_dir):
            for filename in files:
                file_path = os.path.join(root, filename)
```

Reply:
And to clarify - I realise that it wasn't you who wrote this code and decided on those names! 😄 Just noticed it whilst reviewing..